sites/search/search-fr.html

```diff
@@ -0,0 +1,63 @@
+---
+{
+	"altLangPage": "search-fr.html",
+	"dateModified": "2023-06-29",
+	"description": "Documentation sur l'utilisation de la barre de recherche.",
+	"language": "fr",
+	"title": "Exemple pour la Barre de recherche sur un site GC"
+}
+---
+<div class="wb-prettify all-pre hide"></div>
+
+<style>
+	.btn-square {
+		border-radius: 0;
+	}
+</style>
+
+<section class="col-xs-12 col-sm-5 col-md-4">
+	<form action="#" method="post" name="cse-search-box" role="search" class="form-inline">
+		<div class="form-group wb-srch-qry">
+			<label for="wb-srch-q" class="wb-inv">Rechercher dans Canada.ca</label>
+			<div class="input-group">
+				<input id="wb-srch-q" list="wb-srch-q-ac" class="wb-srch-q form-control" name="q" type="search" value="" size="34" maxlength="170" placeholder="Rechercher dans Canada.ca" style="display: inline-block; width: calc(100% - 42px);" />
+				<div class="input-group-append" style="white-space: nowrap;">
+					<button type="submit" class="btn btn-primary btn-square" style="margin-left: -1px;">
+						<span class="glyphicon-search glyphicon"></span>
+						<span class="wb-inv">Search</span>
+					</button>
+				</div>
+			</div>
+			<datalist id="wb-srch-q-ac"></datalist>
+		</div>
+	</form>
+</section>
+
+<br>
+
+<h2>Code</h2>
+<pre><code>
+	&lt;style&gt;
+		.btn-square {
+			border-radius: 0;
+		}
+	&lt;/style&gt;
+
+	&lt;section class="col-xs-12 col-sm-5 col-md-4"&gt;
+		&lt;form action="#" method="post" name="cse-search-box" role="search" class="form-inline"&gt;
+			&lt;div class="form-group wb-srch-qry"&gt;
+				&lt;label for="wb-srch-q" class="wb-inv"&gt;Rechercher dans Canada.ca&lt;/label&gt;
+				&lt;div class="input-group"&gt;
+					&lt;input id="wb-srch-q" list="wb-srch-q-ac" class="wb-srch-q form-control" name="q" type="search" value="" size="34" maxlength="170" placeholder="Rechercher dans Canada.ca" style="display: inline-block; width: calc(100% - 42px);" /&gt;
+					&lt;div class="input-group-append" style="white-space: nowrap;"&gt;
+						&lt;button type="submit" class="btn btn-primary btn-square" style="margin-left: -1px;"&gt;
+							&lt;span class="glyphicon-search glyphicon"&gt;&lt;/span&gt;
+							&lt;span class="wb-inv"&gt;Search&lt;/span&gt;
+						&lt;/button&gt;
+					&lt;/div&gt;
+				&lt;/div&gt;
+				&lt;datalist id="wb-srch-q-ac"&gt;&lt;/datalist&gt;
+			&lt;/div&gt;
+		&lt;/form&gt;
+	&lt;/section&gt;
+</code></pre>
```

Comment:
You need an empty line at the end of the file.

```suggestion
			&lt;/div&gt;
		&lt;/form&gt;
	&lt;/section&gt;
</code></pre>

```